tokio/CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+# 1.6.0 (May 14, 2021)
+
+### Added
+
+- fs: try doing a non-blocking read before punting to the threadpool ([#3518])
```

Comment:
Not critical, but I think this would be a "Changed" and not added (we aren't adding new APIs).